projects/1.18/assets/macaws-fences-biomes-o-plenty/mcwfencesbop/lang/zh_cn.json

```
@@ -1,58 +1,69 @@
-  "block.mcwfencesbop.dead_wired_fence": "枯死木铁丝栅栏",
-  "block.mcwfencesbop.dead_highley_gate": "枯死木栅栏门",
-  "block.mcwfencesbop.fir_picket_fence": "杉木篱栅",
-  "block.mcwfencesbop.fir_stockade_fence": "杉木栅栏墙",
-  "block.mcwfencesbop.fir_horse_fence": "杉木马场栅栏",
-  "block.mcwfencesbop.fir_wired_fence": "杉木铁丝栅栏",
-  "block.mcwfencesbop.fir_highley_gate": "杉木栅栏门",
-  "block.mcwfencesbop.hellbark_picket_fence": "地狱皮木篱栅",
-  "block.mcwfencesbop.hellbark_stockade_fence": "地狱皮木栅栏墙",
-  "block.mcwfencesbop.hellbark_horse_fence": "地狱皮木马场栅栏",
-  "block.mcwfencesbop.hellbark_wired_fence": "地狱皮木铁丝栅栏",
-  "block.mcwfencesbop.hellbark_highley_gate": "地狱皮木栅栏门",
-  "block.mcwfencesbop.jacaranda_picket_fence": "蓝花楹木篱栅",
-  "block.mcwfencesbop.jacaranda_stockade_fence": "蓝花楹木栅栏墙",
-  "block.mcwfencesbop.jacaranda_horse_fence": "蓝花楹木马场栅栏",
-  "block.mcwfencesbop.jacaranda_wired_fence": "蓝花楹木铁丝栅栏",
-  "block.mcwfencesbop.jacaranda_highley_gate": "蓝花楹木栅栏门",
-  "block.mcwfencesbop.magic_picket_fence": "魔法木篱栅",
-  "block.mcwfencesbop.magic_stockade_fence": "魔法木栅栏墙",
-  "block.mcwfencesbop.magic_horse_fence": "魔法木马场栅栏",
-  "block.mcwfencesbop.magic_wired_fence": "魔法木铁丝栅栏",
-  "block.mcwfencesbop.magic_highley_gate": "魔法木栅栏门",
-  "block.mcwfencesbop.mahogany_picket_fence": "桃花心木篱栅",
-  "block.mcwfencesbop.mahogany_stockade_fence": "桃花心木栅栏墙",
-  "block.mcwfencesbop.mahogany_horse_fence": "桃花心木马场栅栏",
-  "block.mcwfencesbop.mahogany_wired_fence": "桃花心木铁丝栅栏",
-  "block.mcwfencesbop.mahogany_highley_gate": "桃花心木栅栏门",
-  "block.mcwfencesbop.palm_picket_fence": "棕榈木篱栅",
-  "block.mcwfencesbop.palm_stockade_fence": "棕榈木栅栏墙",
-  "block.mcwfencesbop.palm_horse_fence": "棕榈木马场栅栏",
-  "block.mcwfencesbop.palm_wired_fence": "棕榈木铁丝栅栏",
-  "block.mcwfencesbop.palm_highley_gate": "棕榈木栅栏门",
-  "block.mcwfencesbop.redwood_picket_fence": "红木篱栅",
-  "block.mcwfencesbop.redwood_stockade_fence": "红木栅栏墙",
-  "block.mcwfencesbop.redwood_horse_fence": "红木马场栅栏",
-  "block.mcwfencesbop.redwood_wired_fence": "红木铁丝栅栏",
-  "block.mcwfencesbop.redwood_highley_gate": "红木栅栏门",
-  "block.mcwfencesbop.umbran_picket_fence": "暗影木篱栅",
-  "block.mcwfencesbop.umbran_stockade_fence": "暗影木栅栏墙",
-  "block.mcwfencesbop.umbran_horse_fence": "暗影木马场栅栏",
-  "block.mcwfencesbop.umbran_wired_fence": "暗影木铁丝栅栏",
-  "block.mcwfencesbop.umbran_highley_gate": "暗影木栅栏门",
-  "block.mcwfencesbop.willow_picket_fence": "柳木篱栅",
-  "block.mcwfencesbop.willow_stockade_fence": "柳木栅栏墙",
-  "block.mcwfencesbop.willow_horse_fence": "柳木马场栅栏",
-  "block.mcwfencesbop.willow_wired_fence": "柳木铁丝栅栏",
-  "block.mcwfencesbop.willow_highley_gate": "柳木栅栏门",
-  "itemGroup.tab_mfbop": "Macaw的栅栏与墙-BOP附属"
-}
\ No newline at end of file
+    "block.mcwfencesbop.cherry_picket_fence": "樱桃木篱栅",
+    "block.mcwfencesbop.cherry_stockade_fence": "樱桃木栅栏墙",
+    "block.mcwfencesbop.cherry_horse_fence": "樱桃木马场栅栏",
+    "block.mcwfencesbop.cherry_wired_fence": "樱桃木铁丝栅栏",
+    "block.mcwfencesbop.cherry_highley_gate": "樱桃木栅栏门",
```

```suggestion
    "block.mcwfencesbop.cherry_highley_gate": "樱桃木海利栅栏门",
```
请将所有`highley_gate`对应的翻译为`海利栅栏门`